org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/SonarLintClientFacade.java

```diff
@@ -78,7 +81,7 @@ private void tryStart() {
       .build();
     try {
       SonarLintCorePlugin.getDefault().info("Starting SonarLint");
-      client.start();
+      client.start(globalConfig);
       this.started = true;
     } catch (Throwable e) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1181)